edx/edx-app-ios
# iOS - Update LTR and RTL translations on mobile

### Description

[LEARNER-7674](https://openedx.atlassian.net/browse/LEARNER-7674)
[LEARNER-7336](https://openedx.atlassian.net/browse/LEARNER-7336)

In this PR I updated all the available translated string for supported LTR and RTL languages in the app from transifex. 

### How to test this PR
- Run the app
- If you are using iOS older then 13 then you can change the device language with Setting-> General -> Language & Region.
- If you are using iOS 13 then you can easily change the app language individually, Setting -> edX app -> Language.
- Re-open the app, all strings should be in the selected language.

Reference URL to change the device language 
https://www.monashfodmap.com/blog/how-change-your-app-language-ios-13/
